CMakeLists.txt

```diff
@@ -560,18 +560,13 @@ IF (ARPACK_FOUND)
 	SET(POSTLINKFLAGS ${POSTLINKFLAGS} ${ARPACK_LIB})
 ENDIF()
 
-OPTION(ENABLE_EIGEN "Enable Eigen" ON)
-OPTION(BUNDLE_EIGEN "Bundle Eigen" OFF)
-IF(BUNDLE_EIGEN)
+OPTION(BUNDLE_EIGEN "Bundle Eigen" ON)
```

Comment:
mmm let's not do this... :)
i.e. remove the whole option `BUNDLE_EIGEN` and when `NOT EIGEN3_FOUND` then just simply include the external eigen...